dojo/templates/dojo/snippets/comments.html

```diff
@@ -20,7 +20,7 @@ <h4>Notes <span class="pull-right"><a data-toggle="collapse" href="#vuln_notes">
   <div class="panel panel-default">
   <div class="panel-comments">
     <strong>{{ note.author.username }}</strong> <span class="text-muted">commented {{ note.date | naturalday }}</span>
-    {% ifequal user.username note.author.username %}
+    {% if user.username == note.author.username or user.is_superuser %}
```

Comment:
Do we need to make this change?

Reply:
It was done to give permission to the admin to delete notes. Thought it will be useful with single entry note types if the admin had permission to do so.